main.cc

```diff
@@ -1280,6 +1280,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             gcfg.shadow_round_ms = cfg->shadow_round_ms();
```

Comment:
Commit message needs expanding, please see my previous comments related to this.